.github/workflows/comptime_ci.yml

```
@@ -0,0 +1,51 @@
+name: Time CI
```

```suggestion
name: Comptime CI
```
Wouldn't that be a more appropriate name?

---

Indeed, copy-pasta error 😄